modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/entity/Detector.java

```diff
@@ -76,4 +77,70 @@
         private Date dateLastUpdated;
     }
 
+    @Data
```

Comment:
Can we use lombok's @AllArgsConstructor
@NoArgsConstructor here to simplify some of the code here?